{
  "uuid": "f4e1aa02-612e-4e56-8745-a0a4db41e481",
  "path": "/usr/lib/python3/dist-packages/setuptools/errors.py",
  "size": 2464,
  "modified": "2023-08-18T12:07:38Z",
  "created": "2025-05-29T10:39:58.977674164Z",
  "sha256_hash": "dae4e8348440edd1bdf79a5ff28c7c6b8afb9c9b4feb6fa1a4b873b118ab9f1d",
  "file_type": "Script",
  "permissions": 33188,
  "is_executable": true,
  "is_signed": false,
  "code_signature": null,
  "bundle_info": null,
  "scan_timestamp": "2025-06-01T18:45:25.491326474Z"
}